src/registry/domain/repository.js

```diff
@@ -230,12 +239,12 @@ module.exports = function(conf){
 
         cdn.putDir(pkgDetails.outputFolder, `${conf.s3.componentsDir}/${componentName}/${componentVersion}`, (err) => {
           if(err){ return callback(err); }
-          componentsCache.refresh(callback);
+          componentsCache.refresh((err, componentsList) => {
+            if(err){ return callback(err); }
+            componentsDetails.refresh(componentsList, callback);
+          });
```

Comment:
On component published, both files are in need to be updated and succeed otherwise the publish will fail and the CLI will error